docker-compose-using-tube-cheese/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+
+services:
+
+  sozu:
+    image: geal/sozu
```

Comment:
Please use this image instead: [clevercloud/sozu](https://hub.docker.com/r/clevercloud/sozu/)